### PR TITLE
Novacustom nv4x/rel 1.2.1

### PR DESCRIPTION
**Pull request by mkopec**

### CODE REVIEW

File: dasharo/releases.yaml [hunk 1]
```diff
@@ -1,4 +1,43 @@
...
+                  url: https://gitlab.com/novacustom/dasharo-compatibility/-/issues/13
+                - description: Laptop cannot output video via the Tunderbolt 4 USB Type-C port
+                  url: https://gitlab.com/novacustom/dasharo-compatibility/-/issues/7
+
+        - version: v1.2.0
+          added:
+                - description: Renamed device to NovaCustom NV4x
+          fixed:
+                - description: Wake from suspend doesn't work with certain SSDs
+                - description: CVE-2022-29264 SMM loader vulnerability
```
Comment (miczyg1) [inline dasharo/releases.yaml:24]: Why listed twice?

**Review by miczyg1 — CHANGES_REQUESTED** [review]

File: dasharo/releases.yaml (hunk 1 above)
Comment (mkopec) [reply dasharo/releases.yaml:24]: Not listed twice, just moved from underneath for correct ordering

Comment (miczyg1) [reply dasharo/releases.yaml:24]: - description: CVE-2022-29264 SMM loader vulnerability
- description: Fix for CVE-2022-29264

It is twice, isn't it?

Comment (mkopec) [reply dasharo/releases.yaml:24]: Ah I see what you mean now. Fixed

File: dasharo/releases.yaml [hunk 2]
```diff
@@ -1,4 +1,42 @@
 releases:
+        - version: v1.2.1
+          fixed:
+                - description: cbfstool logo replacement not working on NV4x v1.2.0
+          known_issues:
+                - description:  Custom fan curve after suspend
```
Comment (miczyg1) [inline dasharo/releases.yaml:6]:
```suggestion
                - description: Custom fan curve after suspend
```

File: dasharo/releases.yaml [hunk 3]
```diff
@@ -1,4 +1,42 @@
...
+
+        - version: v1.2.0
+          added:
+                - description: Renamed device to NovaCustom NV4x
+          fixed:
+                - description: Wake from suspend doesn't work with certain SSDs
+                - description: CVE-2022-29264 SMM loader vulnerability
+                - description: Fix BIOS vendor name in SMBIOS
+          known_issues:
+                - description:  Custom fan curve after suspend
```
Comment (miczyg1) [inline dasharo/releases.yaml:27]:
```suggestion
                - description: Custom fan curve after suspend
```

**Review by miczyg1 — CHANGES_REQUESTED** [review]

**Review by miczyg1 — APPROVED** [review]

**Pull request merged**